src/main/java/de/plushnikov/intellij/plugin/processor/handler/BuilderHandler.java

```diff
@@ -53,7 +53,7 @@
   private static final Collection<String> INVALID_ON_BUILDERS = Collections.unmodifiableSet(new HashSet<String>(Arrays.asList(
     Getter.class.getSimpleName(), Setter.class.getSimpleName(), Wither.class.getSimpleName(), ToString.class.getSimpleName(), EqualsAndHashCode.class.getSimpleName(),
     RequiredArgsConstructor.class.getSimpleName(), AllArgsConstructor.class.getSimpleName(), NoArgsConstructor.class.getSimpleName(),
-    Data.class.getSimpleName(), Value.class.getSimpleName(), lombok.experimental.Value.class.getSimpleName(), FieldDefaults.class.getSimpleName())));
+    Data.class.getSimpleName(), Value.class.getSimpleName(), FieldDefaults.class.getSimpleName())));
 
```

Comment:
In this class the methods:
```
  private boolean isFluentBuilder(@NotNull PsiAnnotation psiAnnotation)
  private boolean isChainBuilder(@NotNull PsiAnnotation psiAnnotation)
``` 
can be removed too.

Reply:
And you can for usages of this methods and simplify it accordingly

Reply:
Also done in commit 2c3c1112fd530827bd8f4c23e57efc2d74258837.